stat.proto

```diff
@@ -120,4 +121,6 @@ message StatJourneySection {
     optional string     to_admin_insee = 29;
 }
 
-
+message StatResponse {
+    required int32 object_count = 1;
```

Comment:
Required field aren't recommended, in theory you won't never be able to remove them, and this feature will probably change over time, so optional seems pretty good